tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -83,13 +83,11 @@ SpatialDiscretization:
     Formulation: StrongInertial
     Quadrature: GaussLobatto
 
+# Note that this test has events chosen so that the AH finder will be the
+# only thing that writes to the reduction data file, so testing for the
+# existence of that file is a weak test of the AH finder working as
+# expected.
```

Comment:
So now ObserveErrorNorms will not be used in the executable?  (That's probably ok if this is the only way that the AH finder can be tested).

Reply:
Yeah, that's right. I felt it would probably be acceptable to anticipate that the `ObserveErrorNorms` is adequately tested by other executables that use the feature.
An alternative could be to have another, similar input file test that does include the `ObserveErrorNorms` if we want to be sure to test that feature with the `EvolveGhKerrSchild` executable.

Reply:
Can you use the `CheckOutputFiles` mechanism to do a weak check on the existence and the content of the reduction data? I think you can use `SkipColumns` to skip any comparisons to zero, or put in some large tolerances, or something like that.

Reply:
Good point. I've rewritten the file to check the `AhA.dat` subfile, and restored the `ObserveErrorNorms`.
Cheers!